inference/core/workflows/core_steps/sinks/modbus_tcp/v1.py

```diff
@@ -0,0 +1,227 @@
+from abc import ABC, abstractmethod
```

Comment:
from what I see this block would not execute properly (returning different outputs than registered)
integration tests will be needed

Reply:
Removed this block.